packages/prisma/extensions/undefined-guard.ts

```diff
@@ -0,0 +1,31 @@
+import { Prisma } from "@prisma/client";
+
+const checkUndefinedInValue = (where: any) => {
+  if (where) {
+    for (const key in where) {
+      // INFO: Since this is for $allModels, we don't have a way to get the correct
+      // where type
+      const whereInput = where[key as any] as any;
+      if (whereInput.hasOwnProperty("in") && whereInput.in === undefined) {
```

Comment:
typeof whereInput.in === "undefined" may be better

Reply:
Why check type versus the exact value?